doc/api/tty.md

```diff
@@ -82,12 +82,15 @@ or `writeStream.rows` properties have changed. No arguments are passed to the
 listener callback when called.
 
 ```js
+process.stdin.setRawMode(true);
```

Comment:
That's correct. (as far as I know) Removed. I don't think it's necessary to mention in commit messages since I both added and removed it?